src/integration/create-express-middleware.js

```diff
@@ -1,16 +1,15 @@
 import express from 'express';
 import request from 'supertest';
-import { createRouteMiddleware } from '../create-route-middleware';
-import { createFeatures } from '../test-fixtures/create-features';
+import { createExpressMiddleware } from '../create-express-middleware';
+import { createFeaturesStringArray } from '../test-fixtures/create-features';
 import { describe } from 'riteway';
 
-describe('createRouteMiddleware()', async should => {
+describe('createExpressMiddleware()', async should => {
   const { assert } = should();
-
   {
     const app = express();
-    const features = createFeatures();
-    const handler = createRouteMiddleware(features, {
+    const features = createFeaturesStringArray();
```

Comment:
don't use `createFeaturesStringArray`. instead set features to an array of feature names. In my opinion, fixtures are good for when you need to test against data that can't easily be defined in each test without cluttering the test, if possible, avoid fixtures because they hide the data from the test definition. This is just my opinion, there could be good counter-arguments.

Reply:
I was questioning whether it would be OK to do it this way. Sounds good though!